cumulusci/tasks/release_notes/task.py

```diff
@@ -55,3 +73,127 @@ def _run_task(self):
+    isn't found, one is created and the BUILD_NOTES_LABEL is applied to it.
+
+    If the BUILD_NOTES_LABEL is present on the pull request, then all notes from the 
+    child pull request are aggregated into the parent pull request. If the BUILD_NOTES_LABEL
+    is not detected on the parent pull request then a link to the child pull request
+    is placed under the "Unaggregated Pull Reqeusts" header.
+
+    When given the parent_branch_name option, this task will query for a corresponding pull request.
+    If a pull request is not found, the task exits. If a pull request is found, then all notes
+    from child pull requests are re-aggregated and the body of the parent is replace entirely.
+    """
+
+    BUILD_NOTES_LABEL = "Build Change Notes"
+
+    task_options = {
+        "branch_name": {"description": "Name of branch with a pull request"},
+        "parent_branch_name": {
+            "description": "name of the parent branch to rebuild change notes for"
+        },
+    }
+
+    def _run_task(self):
+        branch_name = self.options.get("branch_name")
+        parent_branch_name = self.options.get("parent_branch_name")
+
+        if (not branch_name and not parent_branch_name) or (
+            branch_name and parent_branch_name
+        ):
+            raise TaskOptionsError(
+                "You must specify either branch_name or (exclusive) parent_branch_name."
+            )
+
+        self.repo = self.get_repo()
+        generator = ParentPullRequestNotesGenerator(
+            self.github, self.repo, self.project_config
+        )
+
+        if branch_name:
+            self._handle_branch_name_option(generator, branch_name)
+        elif parent_branch_name:
+            self._handle_parent_branch_name_option(generator, parent_branch_name)
+
+    def _handle_branch_name_option(self, generator, branch_name):
+        parent_pull_request = None
+        pull_request = get_pull_request_by_branch_name(self.repo, branch_name)
+
+        if not pull_request:
+            self.logger.info(
+                "Pull request not found for branch: {}.\nExiting...".format(branch_name)
+            )
+            return
+
+        base_branch = pull_request.base.ref
+        feature_prefix = self.project_config.project__git__prefix_feature
+        if not base_branch.startswith(feature_prefix):
+            self.logger.info("Pull request's base is not a feature branch.\nExiting...")
+            return
+
+        parent_pull_request = self._get_parent_pull_request(base_branch)
+        if is_label_on_pull_request(
+            self.repo, parent_pull_request, self.BUILD_NOTES_LABEL
+        ):
+            generator.aggregate_child_change_notes(parent_pull_request)
+        else:
+            generator.update_unaggregated_pr_header(parent_pull_request, branch_name)
+
+    def _handle_parent_branch_name_option(self, generator, parent_branch_name):
+        pull_request = get_pull_request_by_branch_name(self.repo, parent_branch_name)
```

Comment:
As I mentioned above, it's quite likely that there will be multiple pull requests where head is the parent branch. We should look through them for one where base is the repo.default_branch

Reply:
done.